File: Selenium/Activities/Activity16.py
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as chromeService
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
from selenium.webdriver import ActionChains
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
service=chromeService(ChromeDriverManager().install())
with webdriver.Chrome(service=service) as driver:
    actions = webdriver.ActionChains(driver)
    driver.get("https://v1.training-support.net/selenium/dynamic-attributes")
    print(driver.title)

    driver.find_element(By.XPATH,"//input[contains(@class,'-username')]").send_keys("anshul");
    driver.find_element(By.XPATH,"//input[contains(@class,'-password')]").send_keys("password");
    driver.find_element(By.XPATH,"//label[text() = 'Confirm Password']/following-sibling::input").send_keys("password");
    driver.find_element(By.XPATH,"//label[contains(text(), 'mail')]/following-sibling::input").send_keys("abc@gmail.com");
    driver.find_element(By.XPATH,"//button[contains(text(), 'Sign Up')]").click();
    loginMessage = driver.find_element(By.ID,"action-confirmation").text;
    print("Login message: " + loginMessage);
    driver.quit();

File: Selenium/Activities/Activity16.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity16 {
    public static void main(String[] args) {
        WebDriverManager.chromedriver().setup();
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/dynamic-attributes");
        System.out.println(driver.getTitle());
        driver.findElement(By.xpath("//input[contains(@class,'-username')]")).sendKeys("anshul");
        driver.findElement(By.xpath("//input[contains(@class,'-password')]")).sendKeys("password");
        driver.findElement(By.xpath("//label[text() = 'Confirm Password']/following-sibling::input")).sendKeys("password");
        driver.findElement(By.xpath("//label[contains(text(), 'mail')]/following-sibling::input")).sendKeys("abc@gmail.com");
        driver.findElement(By.xpath("//button[contains(text(), 'Sign Up')]")).click();
        String loginMessage = driver.findElement(By.id("action-confirmation")).getText();
        System.out.println("Login message: " + loginMessage);
        driver.quit();
    }
}
